### DISCUSSION TITLE
Maldives 4 outlet hydro jet pump + v2b PCB query

### DISCUSSION THREAD

**Discussion by laconizein** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#669] [Q&A] [answered]
Hi firstly I'm new to all this so please be patient 
I have what I think is a very recent Maldives hydro jet that has 4 outlets not 3 .
I'm including photos of the pump , model number and my v2b PCB.
I've tried and I can get the board to power to the point where I can access the web interface but nothing happens. I can not control the spa either via the web interface or the controls and the controls are completely dead.
I'm confused by the v2b PCB as I know the pin labelling has changed and all the images I see of what wire goes where are for the V2 PCB . 
My questions are as follows a) could someone patiently explain how I wire up the connectors between the spa and the v2b PCB as in what wire goes where b) has anyone got the pump I have working and If so what additional bits do I need? 
Images follow 
![PXL_20240130_223022290 MP](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/20926406/ae39995a-3d88-4917-8b27-aab5127295c2)
![PXL_20240130_223100308 MP](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/20926406/2a2eb0dc-c00b-435a-98c8-4dceab158376)
![PXL_20240130_223244602 MP](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/20926406/450e47d3-571c-4478-b142-df4fb2636ffc)
![PXL_20240130_222758569](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/20926406/b4b77be0-a471-4d2d-a11f-a40da6328ce6)
![PXL_20240130_222815773 MP](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/20926406/c7ebee98-3516-4212-aba8-a9a551ba102b)

**Comment by visualapproach** [comment]
Hi. Seems your level shifter is mounted backwards

**Reply by laconizein** [reply]
lol , thanks for prompt response . still learning and doing this late at night after work probably was not a good idea.

so I don't screw anything else up , one further question 
I have the v2b pcb I look in the guides and I only see Images of the v2 pcb and a v2pcb work around for the newer pumps.
that has confused me a little. 
For the two connectors , between pump and display  which wires go where and what way up are the connectors meant to be.
is it as simple as just following the photos and plugging them in as if it was a v2pcb or is there a different wiring scheme?

**Reply by visualapproach** [reply]
I recommend that you connect your cables stumps to the pump. Then find out with a DMM which wire has 5V and GND. Then solder it accordingly to the pcb.

**Reply by laconizein** [reply]
Ok this is an improvement. Full control of pump by the web!
No display or buttons though I've found I can just plug display into the headers I've soldered on the board
![17067472014255830653451814016035](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/20926406/665f195f-3e62-4dc8-83eb-c6aae0f4abde)
What next?

**Comment by laconizein** [comment, marked as answer]
![PXL_20240203_193740338 MP](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/20926406/b85da808-64cd-4c6e-b247-ff94ed98264f)
Got it working at the resistor's and capacitors.

Odd thing was that the buttons did not work as well as the LCD display.  I'd not come across that in my trawling of the forum
